magenta-lib/src/main/scala/magenta/deployment_type/AutoScaling.scala

```diff
@@ -123,6 +123,7 @@ object AutoScaling  extends DeploymentType with S3AclParams {
         )
       List(
         S3Upload(
+          target.region,
```

Comment:
I can see arguments both ways. If you're trying to be across multiple regions for resiliency, then having the artifact in both regions seems like a good thing.